.gitignore

```diff
@@ -144,3 +145,4 @@ gradle-app.setting
 
 support/
```

Comment:
is this ignoring that src/test/resources/support/ described on the readme? we shouldn't ignore that one! it might include schemas or other useful files used on some features